data-access/nexustiles/nexustiles.py

```diff
@@ -47,6 +49,10 @@ class NexusTileServiceException(Exception):
 
 class NexusTileService(object):
     def __init__(self, skipCassandra=False, skipSolr=False, config=None):
+        self._cass = None
```

Comment:
This could be cleaned up further. NexusTileService could be refactored to take a 'datastore' and a 'metadatastore' object. Both could be optional. This would eliminate the need for the conditional logic later.